libpod/define/version.go

```diff
@@ -17,6 +18,16 @@ var (
 	buildInfo string
 )
 
+// humanVersion is an human output struct for varlink
+type humanVersion struct {
```

Comment:
Why is this private? Why not `HumanVersion`, since this is used outside of Libpod?

Reply:
The goal was to not expose it outside of libpod.  Just to give a prettier view to users in the podman info call.

Reply:
I think it does make it to the remote API, though.

Reply:
Looks like it does - I see calls to it in the APIv2 handlers

Reply:
Another option would be to just return both BuiltTime and Built so we could have human form and epoch seconds.